integration_tests/integration/bookedtoday/arrivals/autoMatchingRecords/hasMultipleMatches.cy.ts

```diff
@@ -73,7 +73,10 @@ context('Arrival matches multiple records', () => {
       match.fieldName('prison-number').should('contain', arrival.potentialMatches[0].prisonNumber)
       match.fieldName('pnc-number').should('contain', arrival.potentialMatches[0].pncNumber)
       match.fieldName('cro-number').should('contain', arrival.potentialMatches[0].croNumber)
-      match.prisonerImage().should('have.attr', 'src', `/prisoners/${arrival.potentialMatches[0].prisonNumber}/image`)
+      match.prisonerImage.check({
```

Comment:
Most of the changes here are moving to use a new utility to test image attrributes